### PR TITLE
Connecting client info page

### PR DESCRIPTION
**Pull request by leenlab2**
Clicking on the "View Client Info" button now goes to an input form, which redirects to the specific client's information page.

## Context

We had the input form connected previously, but it did not take you anywhere when you input a client ID. Now it connects to the intended destination. Related to issue #19 

## Your Changes

**Description**:

- Updated routes so that clicking the action button routes to the input form which routes to a client page
- Made client info container more abstract by creating a new component that has the actual details about the client
- fixed the resulting bugs

**Type of change** (select all that apply):

- [x] Breaking change (changing code functionality)
- [x] Bug fix (non-breaking change that solves an error/issue)

## Testing

When runngin locally, following the path described above should lead to a page containing "last name, first name" with a url matching the client id

## Notes

i will update the tests once i finish implementing the actual client info page

## Checklist

- [x] Code is up to date with main branch (no merge conflicts)
- [x] Code has been properly tested
- [x] Code has been properly documented
- [x] One or two people have reviewed this pull request

### CODE REVIEW

**Comment by leenlab2** [comment]
[connect client information page](https://trello.com/c/39AYVq6d/120-connect-client-information-page)

**Review by dhruvaa02 — COMMENTED** [review]
Why are there hard coded account ID strings rather than taking in some input or something? Or is it some form of test?

**Comment by dhruvaa02** [comment]
Also, the code is still using the IP address for our backend logic, should be the cb.caravantage domain thing instead

File: src/components/clientInfo/ClientInfo.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import CarList from "./CarList";
+import CarDetails from "./CarDetails";
+import {useParams} from "react-router-dom";
+
+export default function ClientInfo(props) {
+    let params = useParams();
+    const clientName = "Last Name, First Name";
+    const currentCarID = "10001";
```
Comment (Infinityworld13) [inline src/components/clientInfo/ClientInfo.js:9]: We should probably change this hardcoded data into inputs for the functions and classes

File: src/components/clientInfo/CarList.js [hunk 2]
```diff
@@ -4,6 +4,9 @@ import ListItemCar from "./ListItemCar";
 export default class CarList extends React.Component {
```
Comment (Infinityworld13) [inline src/components/clientInfo/CarList.js:4]: The class and functions of CarList are missing documentation overall

**Review by Infinityworld13 — COMMENTED** [review]
Apart from a couple of comments for minor things like documentation it works from a separate computer and seems to be in working order.

**Comment by leenlab2** [comment]
> Why are there hard coded account ID strings rather than taking in some input or something? Or is it some form of test?

This PR was focused on getting things connected to each other, the next one will remove the hardcoding and make everything functional (I made a separate PR since the latter will involve a lot of refactoring)

File: src/components/clientInfo/ClientInfo.js (hunk 1 above)
Comment (leenlab2) [reply src/components/clientInfo/ClientInfo.js:9]: Yes, that is going in the next PR just to keep changes small

File: src/components/clientInfo/CarList.js (hunk 2 above)
Comment (leenlab2) [reply src/components/clientInfo/CarList.js:4]: Since the functions are not actually implemented yet, I will add the documentation when I implement them

**Pull request merged**